bears/vcs/git/GitCommitBear.py

```diff
@@ -24,4 +25,32 @@ def get_remotes():
 
     def get_head_commit(self):
         with change_directory(self.get_config_dir() or os.getcwd()):
+            command = self.github_pull_request_temporary_merge_commit()
+            if command:
+                return run_shell_command(command)
+
             return run_shell_command('git log -1 --pretty=%B')
+
+    def github_pull_request_temporary_merge_commit(self):
+        """
+        This detects the merge commit generated by Travis in
+        a fast-forward PR. For more details visit:
+        https://github.com/travis-ci/travis-ci/issues/8400
+        """
+        stdout, stderr = run_shell_command('git log -1 --pretty=%B')
+
+        pos = stdout.find('\n')
+        shortlog = stdout[:pos] if pos != -1 else stdout
```

Comment:
The git commit shortlog is always placed in the first line of the message, so I think `shortlog = stdout.splitlines()[0]` will be good enough..

Reply:
Gives error this way

Reply:
what is the error?
show the commit where you tried to do this, and the travis log of it failing.